### PR TITLE
Updated singleton providers to use providedIn: root

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/icons/icon-manifest.module.ts [hunk 1]
```diff
@@ -2,13 +2,5 @@ import {
...
-  SkyThemeIconManifestService
-} from './icon-manifest.service';
-
-@NgModule({
-  providers: [
-    SkyThemeIconManifestService
-  ]
-})
+@NgModule()
 export class SkyThemeIconManifestModule { }
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/icons/icon-manifest.module.ts:6]: ☝️ Do we need this module anymore?

Comment (Blackbaud-SteveBrush) [reply src/app/public/icons/icon-manifest.module.ts:6]: We should probably keep it in for backwards compatibility, but add a deprecated message above it saying it's no longer needed and can be removed.

Comment (Blackbaud-AlexKingman) [reply src/app/public/icons/icon-manifest.module.ts:10]: Done.

File: src/app/public/icons/icon-manifest.module.ts [hunk 2]
```diff
@@ -2,13 +2,9 @@ import {
...
-} from './icon-manifest.service';
-
-@NgModule({
-  providers: [
-    SkyThemeIconManifestService
-  ]
-})
+/**
+ * @deprecated `SkyThemeIconManifestModule` is no longer needed
+ * and can be removed from your providers.
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/icons/icon-manifest.module.ts:7]: "...and can be removed from your project." (They wouldn't be importing the module into their "providers" array?)

**Pull request closed without merging**